tests/test-extensions/marvel-extension/src/main/java/org/mule/test/marvel/drstrange/DrStrangeOperations.java

```diff
@@ -41,6 +46,8 @@
 
 public class DrStrangeOperations {
 
+  private static int HUGH_SIZE = InMemoryCursorStreamConfig.getDefault().getMaxBufferSize().toBytes() + 1;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Rename this field "HUGH_SIZE" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. ([squid:S3008](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3AS3008))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-runtime-mule&pullRequest=7751&issues=AWm78GvHotIXnmsUmYjL&open=AWm78GvHotIXnmsUmYjL)

[ISSUE_KEY]: AWm78GvHotIXnmsUmYjL